packages/core/src/modules/dids/services/DidResolverService.ts

```diff
@@ -59,6 +60,18 @@ export class DidResolverService {
     return resolver.resolve(parsed.did, parsed, options)
   }
 
+  public async resolveDidDocument(did: string) {
+    const {
+      didDocument,
+      didResolutionMetadata: { error, message },
+    } = await this.resolve(did)
+
+    if (!didDocument) {
+      throw new AriesFrameworkError(`Unable to resolve did document for did '${did}': ${error} ${message}`)
+    }
+    return didDocument
+  }
```

Comment:
Hmm I'm not sure if `resolveDidDocument` is a good name in comparison to `resolve`?

Reply:
Definitely not ideal. Any ideas?